src/recon.rs

```diff
@@ -3935,7 +3927,7 @@ pub(crate) unsafe fn rav1d_recon_b_inter<BD: BitDepth>(
                 y += ytx.h as c_int;
                 y_off += 1;
             }
-            y_dst += y_dst.pixel_stride::<BD>() * 4 * y as isize;
+            y_dst -= y_dst.pixel_stride::<BD>() * 4 * y as isize;
```

Comment:
Why did this change from `+=` to `-=`? This looks incorrect to me, but I might be missing something.

Reply:
It was a bug.

Reply:
You can see in the `dst.offset` version below that there's `-`.